style.css

```diff
@@ -0,0 +1,72 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+/* the nav bar */
+nav {
+  display: flex;
+  justify-content: space-between;
+  background-color: white;
+  padding: 12px 30px;
+}
+
+/* logo */
+.logo h1 {
+  color: #6070ff;
+  font-weight: 600;
+  font-size: 24px;
+}
+
+/* toggle-menu */
+.toggle-menu img {
+  width: 28px;
+}
+
+/* INTRODUCTION SECTION */
+.intro-section {
+  background: url(./assets/images/header-bg-mobile.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 80vh;
+  padding-left: 40px;
+  text-align: left;
+}
+
+/* HEADLINE */
+.headline {
+  font-size: 40px;
+  line-height: 54px;
+  font-weight: 800;
+  padding: 10px 0;
+  margin-top: 10px;
+  margin-right: 10px;
+  padding-top: 100px;
+}
+
+/* DESCRIPTION */
+.description {
+  margin-top: 10px;
+  width: 300px;
+  font-size: 18px;
+  margin-bottom: 10px;
+  font-weight: 400;
+}
+
+/* SOCIAL MEDIA */
+.social-handles p {
+  color: #7f8cff;
+  font-size: 20px;
+  margin-bottom: 10px;
+}
+
+.social-icons {
+  list-style: none;
+  display: flex;
+}
+
+.social-icons img {
+  width: 24px;
+  margin-right: 20px;
+}
```

Comment:
- [ **OPTIONAL** ] Please, kindly make sure the whole design match perfectly the one from [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). You should use the correct:
   - Colors
   - Fonts, Size, weight, height,...
   - Spacing between elements.
   - Etc
   
Stick your design as well as possible to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design.
 ## Figma Design:
![Figma](https://user-images.githubusercontent.com/62290545/170314420-8dbf3006-4360-4a2c-b162-20f367b00995.png)

## Your design
![st](https://user-images.githubusercontent.com/83042742/170456294-1ae27eb6-bec2-4ce4-94a4-a6697d793521.png)